components/binary_sensor/ble_presence.rst

```diff
@@ -22,13 +22,25 @@ bluetooth low energy device.
       - platform: ble_presence
         mac_address: AC:37:43:77:5F:4C
         name: "ESP32 BLE Tracker Google Home Mini"
+      - platform: ble_presence
```

Comment:
1. Please only add one example for service_uuid here. The example at the top should always be minimal to give the user an idea. You can create a `.. note::`` below with more examples with different uuid lengths.

2. Please put a comment above each of these two entries (mac and uuid). One saying `# Presence based on MAC address` and the other the same but with UUID.